INSTALL

```diff
@@ -668,22 +668,33 @@
      ("openssl"). The libraries will be built in the top-level directory,
      and the binary will be in the "apps" subdirectory.
 
+     Troubleshooting hints:
```

Comment:
How about omitting naming the section just "Troubleshooting" instead of "Troubleshooting hints"?